tests/integration/testlib/vxlan.py

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+from contextlib import contextmanager
+
+import libnmstate
+
+from libnmstate.schema import VXLAN
+from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceState
+
+VXLAN_IFNAME = '{}.{}'
```

Comment:
It is unclear why you need this one. It is used once and has no special value as far as I can understand.
(BTW, it is a name template, the current name is misleading)